packages/server/tests/acceptance/htsPrecompile.spec.ts

```diff
@@ -361,4 +361,50 @@ describe('HTS Precompile Acceptance Tests', async function () {
       expect(pauseStatusAfter).to.equal(false);
     });
   });
+
+  describe('HTS Precompile Default Status Tests', async function() {
+    it('should be able to get default freeze status for fungible token', async () => {
+      const baseHTSContract = new ethers.Contract(BaseHTSContractAddress, BaseHTSJson.abi, accounts[0].wallet);
+      
+      const txTokenDefaultFreezeStatus = await baseHTSContract.getTokenDefaultFreezeStatusPublic(HTSTokenContractAddress, {gasLimit: 1000000});
```

Comment:
nit: These 5 lines can be share by both the freeze methods

Reply:
I've extended the existing freeze tests to implement this default status method.